README.md

```diff
@@ -11,6 +13,28 @@ part of the array each recursive call considers.
 
 ## Runtime Analysis
 
-Analyse the time complexity of your implementation and give a $\Theta$ bound for
-its worst-case runtime. Add your answer, including your reasoning, to this
-markdown file.
+`Outer Loop`:
+Increases the size of the subarrays to be merged, starting from 1 and doubling each time until it reaches the length of the array ($n$). The loop runs $\log(n)$ times because with each iteration the size of subarrays doubles (size = 1, 2, 4, ..., n).
+
+`Inner Loop`:
+For a given size, the loop iterates approximately $\frac{n}{2 \cdot size}$ times because each merge operation handles $2 \cdot size$ elements and merging the two.
+
+`Merge`:
+The most expensive operation within the merge function is the process of _shifting_ elements into a correct position. When an element from the right subarray is moved to a position in the left, all subsequent elements in the left subarray need to be shifted right to make space. This operation has a complexity of up to $\Theta(n)$ in the worst case for a single merge step, as it may involve shifting all of the array over to its new location.
```

Comment:
...so what's the overall complexity of merge?

Reply:
Oh. It's n^2 not n. I will update the analysis.